ext/hal/st/stm32cube/stm32f4xx/drivers/include/stm32f4xx_hal_eth.h

```diff
@@ -624,6 +624,17 @@ typedef struct
   * @brief  ETH Handle Structure definition
   */
 
+/*ptp TimeStam struct*/
```

Comment:
This is a nice addition, but unfortunately, it is not allowed to this kind of modifications inside ext/hal/.
Reason is this is a complete new feature and it won't be maintained in this package nor deployed in other Cube packages.
I suggest you to find another way to add the functionality in zephyr ethernet driver and submit a request to ST to implement this feature in STM32Cube packages, you can request this here: https://community.st.com/s/topic/0TO0X000000BSqSWAW/stm32-mcus. I could do it, but I think it will get more impact if coming from customer.

Reply:
I left a message on the st website. Unfortunately, I saw the response from the st employee，“Currently, ST doesn't provide the PTP feature. I will raise your request to our development team to implement this feature. But, unfortunately is not at the short term planning.”

Reply:
Ok, well. At least we know the status, and you've informed that some people are interested in this feature.